src/imports/Components/Popups/1.3/PopupBase.qml

```diff
@@ -142,6 +142,14 @@ OrientationHelper {
       */
     property alias __dimBackground: background.dim
 
+    /*!
+      \internal
+      Property driving the dim color popup's background. The default is the same as
+      defined in the style
+      */
+    property alias __dimBackgroundColor: background.color
```

Comment:
Why not just `__backgroundColor`?

Reply:
humm maybe to see difference between the dim background and the content bg ?.

Reply:
Ah, OK, I didn't know we were having two different backgrounds.